cdap-docs/examples-manual/source/examples/user-profiles.rst

```diff
@@ -0,0 +1,245 @@
+.. meta::
```

Comment:
I am assuming that you left this unchanged except for converting it to .rst and moving it here?